py/kubeflow/testing/cleanup_ci.py

```diff
@@ -47,6 +47,9 @@
 # resource. So we we just associae a long max age with this.
 E2E_OWNERLESS = "e2e-ownerless"
 
+DEFAULT_ZONES = ("us-east1-b,us-east1-c,us-east1-d,us-central1-a,"
```

Comment:
Minor: Originally it only has `"us-east1-d,us-central1-a` and it has more zones now. We may move to arguments with default in the future to avoid potential changes.